Python/Product/PythonTools/PythonTools.vsct

```diff
@@ -624,8 +624,17 @@ permissions and limitations under the License.
         </Strings>
       </Button>
 
+      <Button guid="guidPythonToolsCmdSet" id="cmdidProcessRequirementsTxt" priority="0x0610" type="Button">
+        <CommandFlag>AllowParams</CommandFlag>
+        <CommandFlag>CommandWellOnly</CommandFlag>
+        <Strings>
+          <ButtonText>Pro&amp;cess Requirements</ButtonText>
```

Comment:
I didn't like the name either, we can use your suggestion.  It may require changing some strings to be a bit more generic, but reusing the command from the wizard may be possible.  Let me see.